libs/tickets/src/lib/tickets/tickets.component.ts

```diff
@@ -58,4 +60,7 @@ export class TicketsComponent implements OnInit {
       }
     ];
   }
+  paginateTable($event?: any): void {
+    this.TicketsService.getTickets($event);
```

Comment:
Instead of using the getTickets service use the facade. Allow for optional params to be passed into the init facade for pagination